opennms-doc/guide-install/src/asciidoc/text/newts/setup-newts.adoc

```diff
@@ -0,0 +1,48 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[gi-setup-newts]]
+=== Setting up Newts
+
+The setup for _Newts_ has two parts, the first one initializes the keyspace in the _Cassandra_ storage and the second part is the standalone running _Newts API_.
+Both parts are packaged and maintained in the link:https://github.com/OpenNMS/newts/releases[Newts GitHub repository].
+The latest pre-compiled tarball can be downloaded as link:https://github.com/OpenNMS/newts/releases/download/1.3.0/newts-1.3.0-bin.tar.gz[newts-1.3.0-bin.tar.gz]
```

Comment:
We need to change this for each newts-release? Isn't there something like newts-LATEST-bin.tar.gz?